Tests/MusicXMLTests/ScoreTests/README.md

```diff
@@ -0,0 +1,28 @@
+# ScoreTests
+
+## Test Case Structure
+The score tests consist of test suite executor and test cases. Any individual test case is a folder that satisfy the following rules.
+
+- The folder name is arbitrary
+- The folder must contain a `manifest.plist` file, otherwise the folder is not treated as a test case and thus skipped.
+
+### Manifest
+The `manifest.plist` further defines the test case.
+- The manifest is a dictionary at its root.
+- The manifest MUST contain a `DisplayName` String field to specify the test name.
```

Comment:
I think that **must** might be easier on the eyes. Particularly in concert with below.